### PR TITLE
Update contract layout SMTChecker

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: docs/layout-of-source-files.rst [hunk 1]
```diff
@@ -86,8 +86,10 @@ This component has to be enabled when the Solidity compiler is built
 and therefore it is not available in all Solidity binaries.
 The :ref:`build instructions<smt_solvers_build>` explain how to activate this option.
 It is activated for the Ubuntu PPA releases in most versions,
-but not for solc-js, the Docker images, Windows binaries or the
-statically-built Linux binaries.
+but not for the Docker images, Windows binaries or the
+statically-built Linux binaries. It can be activated for solc-js via the
+`smtCallback <https://github.com/ethereum/solc-js>`_ if you have an SMT solver
```
Comment (chriseth) [inline docs/layout-of-source-files.rst]: Is it automatic?

Comment (chriseth) [reply docs/layout-of-source-files.rst]: Maybe link https://github.com/ethereum/solc-js#example-usage-with-smtsolver-callback ?

Comment (leonardoalt) [reply docs/layout-of-source-files.rst]: not automatic, wanted to link to the example

Comment (leonardoalt) [reply docs/layout-of-source-files.rst]: updated

**Review by chriseth — APPROVED** [review]

**Pull request merged**